swagger/v2/swagger.yaml

```diff
@@ -675,6 +675,21 @@ paths:
             to get a directory tree.
           required: false
           type: boolean
+        - name: cr_identifier
+          in: query
+          description: |
+            Identifier of a catalog record. browse only files that have been selected for that record.
```

Comment:
Should it mention here that also internal pk is a valid value for these parameter? It might be useful for service users and why not for endusers as well